winml/test/api/LearningModelAPITest.cpp

```diff
@@ -37,6 +37,18 @@ static void CreateModelFileNotFound() {
     });
 }
 
+static void CreateCorruptModel() {
+  LearningModel learningModel = nullptr;
+
+  WINML_EXPECT_THROW_SPECIFIC(
+    APITest::LoadModel(L"corrupt-model.onnx", learningModel),
```

Comment:
Can this model be generated on the fly (null stream) rather than adding new collateral?